### DISCUSSION TITLE
求教大佬，R2S软路由应该下载哪个文件进行v3.11的更新？

### DISCUSSION THREAD

**Discussion by da-le-si-jie** [AlistGo/alist#3418] [Q&A]
<img width="360" alt="image" src="https://user-images.githubusercontent.com/123718068/218911702-f525127a-70f2-4b78-bce4-04846bf83c02.png">

先后下载了多个版本，到openwrt界面发现都不能更新……

<img width="1202" alt="image" src="https://user-images.githubusercontent.com/123718068/218911801-da087ef6-b086-4b34-bb09-9586f1674d58.png">

**Comment by weekstarR2S** [comment]
* openwrt安装alist
rf https://github.com/sbwml/luci-app-alist

 修改清华源
 刷新软件源
opkg update

 安装curl
opkg install curl

 安装alist
*** sh -c "$(curl -ksS https://raw.githubusercontent.com/sbwml/luci-
app-alist/master/install.sh)"
*** log
Checking available space
空间

Checking platform  ...
平台

Update opkg sources
更新软件源

Download Packages ...
Downloading https://github.com/sbwml/luci-app-alist/releases/download/v3.6.0/alist_3.6.0-1_x86_64.ipk
alist

Downloading https://github.com/sbwml/luci-app-alist/releases/download/v3.6.0/luci-app-alist_1.0.7_all.ipk
luci图形配置

Downloading https://github.com/sbwml/luci-app-alist/releases/download/v3.6.0/luci-i18n-alist-zh-cn_git-22.315.57872-eca2b61_all.ipk
中文语言包

Install Packages ...

Installing alist (3.6.0-1) to root...
Configuring alist.
Installing luci-app-alist (1.0.7) to root...
Configuring luci-app-alist.
Installing luci-i18n-alist-zh-cn (git-22.315.57872-eca2b61) to root...
Configuring luci-i18n-alist-zh-cn.
 国内镜像
github被墙

 查看密码
alist --force-bin-dir --debug admin

 启动
alist --force-bin-dir --debug server &

 首次登录
改密码
改语言
